Casks/scenica-player.rb

```diff
@@ -0,0 +1,20 @@
+cask "scenica-player" do
+  version "1.3.2"
+  sha256 "59d7417cf3801de0db32d44694eb670a03410fdd206f1df90f8e113d4872ee0b"
+
+  url "https://sceni.ca/download/scenica-player/scenica-player-#{version}.zip"
+  name "Scenica Player"
+  desc "Turn your device into an on-set player"
+  homepage "https://sceni.ca/en/player/"
+
+  livecheck do
+    url :homepage
+    regex(/href=.*?scenica-player[._-]v?(\d+(?:\.\d+)+)\.zip/i)
+  end
+
+  app "Scenica Player.app"
+
+  zap trash: [
+    "~/Library/Preferences/ByHost/ca.sceni.Player.B97E4AB2-D53B-5F4B-8160-0481CF87A038.plist",
```

Comment:
```suggestion
    "~/Library/Preferences/ByHost/ca.sceni.Player.*.plist",
```

This number looks like it could be a UUID, so we can use a wildcard.